MapboxVision/VisionManager/BaseVisionManager.swift

```diff
@@ -3,11 +3,24 @@ import Foundation
 public class BaseVisionManager: VisionManagerProtocol {
     // MARK: Performance control
 
+    public var modelPerformance = ModelPerformance(mode: .dynamic, rate: .high) {
+        didSet {
+            guard oldValue != modelPerformance else { return }
+            updateModelPerformance(modelPerformance)
+        }
+    }
+
+    @available(*, deprecated, message: "Use ModelPerformance structure instead. Enum will be removed in version 0.13.0")
```

Comment:
nit:

A bit more clarity to avoid the possible confusion?

```suggestion
    @available(*, deprecated, message: "Use ModelPerformance structure instead. ModelPerformanceConfig will be removed in version 0.13.0")
```

or

```suggestion
    @available(*, deprecated, message: "Use ModelPerformance structure instead. Property will be removed in version 0.13.0")
```

?

Reply:
If you decide to update message, could you please the rest of diff.

Reply:
Adjusted, PTAL.